src/scripts/main.js

```diff
@@ -1 +1,21 @@
 'use strict';
+
+const bigImg = document.getElementById('largeImg');
+
+const thumbs = document.getElementById('thumbs');
+
+thumbs.addEventListener('click', (e) => {
+  const list = e.target.closest('.list-item');
+
+  if (!list) {
+    return;
+  }
+
+  e.preventDefault();
+
+  const link = list.querySelector('.list-item__link');
+
+  if (link) {
+    bigImg.src = link.href;
```

Comment:
Ensure that `link.href` points directly to an image URL. If `link.href` is not a direct image URL, consider extracting the image URL from a different attribute or element.